app/components/bali/progress/component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Bali
+  module Progress
+    class Component < ApplicationViewComponent
+      attr_reader :value, :display_percentage, :color_code, :options
+
+      def initialize(value: 50, display_percentage: true, color_code: default_color, **options)
+        @value = value
+        @display_percentage = display_percentage
+
+        @progress_bar_color = progress_bar_color(color_code)
+        @options = prepend_class_name(options, 'progress-component')
```

Comment:
Creo que podemos simplificar los nombres, sobre todo no incluir progress, ya que estamos dentro del progress component

```suggestion
      def initialize(value: 50, percentage: true, color: default_color, **options)
        @value = value
        @percentage = percentage

        @bar_color = bar_color(color)
        @options = prepend_class_name(options, 'progress-component')
```